snmp.yml

```diff
@@ -4636,7 +4636,6 @@ cisco_wlc:
     indexes:
     - labelname: bsnAPDot3MacAddress
       type: PhysAddress48
-      fixed_size: 6
```

Comment:
It looks like your generator or mibs are out of date. These should not be removed.

Reply:
not sure - I've rebased the latest main branch of this repo and the mibs were just generated with make mibs.

any ideas?